docs/api.md

```diff
@@ -25,14 +25,17 @@ will overwrite them.
 
 * `password` `(string: <required>)` - OpenStack password of the root user.
 
+* `project_name` `(string: <required>)` - Name of the project of the root user.
```

Comment:
Is it really required?

Reply:
Also, to reduce phrase complexity, you can put at as `Project name of the root user`